components/CardCarousel.jsx

```diff
@@ -0,0 +1,53 @@
+import styled from "styled-components";
+import { Swiper, SwiperSlide } from "swiper/react";
+import SwiperCore, { Pagination } from "swiper/core";
+
+import { Card } from "./Card";
+import { media } from "../utils";
+
+SwiperCore.use([Pagination]);
+
+export function CardCarousel({ slides, ...props }) {
+  return (
+    <div {...props}>
+      <Swiper
+        slidesPerView={1}
+        spaceBetween={10}
+        pagination={{
+          clickable: true,
+        }}
+        breakpoints={{
+          1200: {
+            slidesPerView: 3,
+            spaceBetween: 20,
+          },
+          750: {
+            slidesPerView: 2,
+            spaceBetween: 20,
```

Comment:
(For 1. from the review comment, add more slidesPerView as the screen gets wider): 
- continue increasing slidesPerView relative to the screen width (eg. 1500-1600 for 4 per view, etc) 
  - might be nice to keep the width of the carousel at 100% if there's an easy way to keep increasing slidesPerView as screen increases